meta-balena-common/recipes-support/resin-init/resin-init-flasher/resin-init-flasher

```diff
@@ -131,29 +138,31 @@ else
 fi
 inform "balena configuration found."
 
+# BALENA_BOOT_MOUNTPOINT should exist
+if [ ! -d "$BALENA_BOOT_MOUNTPOINT" ]; then
+	fail "$BALENA_BOOT_MOUNTPOINT : No such directory."
+fi
+
 # CONFIG_PATH should exist
 if [ ! -f "$CONFIG_PATH" ]; then
     fail "$CONFIG_PATH : No such file."
 fi
 
-# BALENA_BOOT_MOUNTPOINT should exit
-if [ ! -d "$BALENA_BOOT_MOUNTPOINT" ]; then
-	fail "$BALENA_BOOT_MOUNTPOINT : No such directory."
-fi
-
 # Give a chance to openvpn to come alive
 STARTTIME=$(date +%s)
 ENDTIME="$STARTTIME"
 TIMEOUT=15
-while [ "$(systemctl is-active openvpn)" != "active" ]
-do
-    if [ $((ENDTIME - STARTTIME)) -le $TIMEOUT ]; then
-        sleep 1 && ENDTIME=$((ENDTIME + 1))
-    else
-        inform "Timeout while waiting for openvpn to come alive. No network?"
-        break
-    fi
-done
+if command -v "systemctl"; then
```

Comment:
Just for my understanding - the initrd will not try to initialize VPN or register the device with the cloud?

Reply:
No, the initrd has no way to bring up things like cellular or wifi or static network configurations, so cloud connectivity would be an inconsistent feature that would only work in a subset of devices with local dynamic IP configuration. Even now that theoretically we can debug flasher devices on the field I have never seen it being done.